cloud/ec2.go

```diff
@@ -426,6 +427,13 @@ func (m *ec2Manager) spawnOnDemandHost(ctx context.Context, h *host.Host, ec2Set
 			"host_provider": h.Distro.Provider,
 			"distro":        h.Distro.Id,
 		}))
+		detailErr := task.AddHostCreateDetails(h.StartedBy, h.Id, h.SpawnOptions.TaskExecutionNumber, err)
+		grip.ErrorWhen(detailErr != nil, message.WrapError(detailErr, message.Fields{
+			"message":       "adding host create error details",
```

Comment:
For splunk messages, I think it makes sense for these to be full sentences since they aren't wrapped errors (added a suggestion to the convention doc)

Reply:
done